src/scripts/party/Party.ts

```diff
@@ -174,7 +174,7 @@ class Party implements Feature {
 
     public calculateEVAttackBonus(pokemon: PartyPokemon): number {
         const EVs = this.getEffortValues(pokemon)();
-        if (!pokemon.pokerus) {
+        if (pokemon.pokerus < GameConstants.Pokerus.Contagious) {
```

Comment:
I'm not sure I get this part, shouldn't they be able to get EVs as soon as they are infected, even if not contagious?

Reply:
We could make it that way. I don't think it matters very much. A Pokémon is ``Infected`` only for a short period of time before they hatch, when they then become contagious. You could theoretically catch that Pokémon in that time when it is in the hatchery and ``Infected`` (but not ``Contagious``), but I figured it was better this way.

Reply:
👍 Ah I see, works fine this way, I was just mistaken in how it all worked 😅